pkg/microservice/warpdrive/core/service/taskplugin/job.go

```diff
@@ -226,6 +226,16 @@ func (b *JobCtxBuilder) BuildReaperContext(pipelineTask *task.Task, serviceName
 		ServiceName:     serviceName,
 		StorageEndpoint: pipelineTask.StorageEndpoint,
 		AesKey:          pipelineTask.ConfigPayload.AesKey,
+		UploadEnabled:   b.JobCtx.UploadEnabled,
+		UploadStorageInfo: &task.ObjectStorageInfo{
+			Endpoint: b.JobCtx.UploadStorageInfo.Endpoint,
```

Comment:
Will it be possible that `b.JobCtx.UploadStorageInfo` is `nil`?

Reply:
It won't